Jenkinsfile.cloud

```diff
@@ -216,7 +216,7 @@ node(NODE) {
                 string(credentialsId: params.AWS_CI_ACCOUNT, variable: 'AWS_CI_ACCOUNT'),
             ]) {
                 sh """
-                    amijson=${dirpath}/aws-${AWS_REGION}-smoketested.json
+                    amijson=${dirpath}/aws-${AWS_REGION}.json
```

Comment:
Hmm, we decided to add the `smoketested` tag since we run `rhcos.basic` tests in the cloud job before we generate the ami, thus the generated image should at least boot, can ssh, etc. I'm ok with changing it, but that's the original reasoning we had behind it.

Reply:
Since we don't upload it at all if it doesn't pass the smoketest, I think I'm OK with omitting it. Kinda bikeshed though, I'm fine keeping it if someone feels strongly about it.